requirements.txt

```diff
@@ -1,5 +1,4 @@
 # Django/Framework Packages
-django>=1.8,<1.9
```

Comment:
I don't think we want to fully remove this, as this repository's code is intended to be used in non-tox environments in production and the requirement needs to be specified somewhere.

For example, `django-config-models` specifies a base version [here](https://github.com/edx/django-config-models/blob/master/requirements/base.txt#L7), then has [tox overwrite the base-installed version here](https://github.com/edx/django-config-models/blob/master/tox.ini#L19-L22).

I'm not sure what the correct version to specify here is though - should it be `django>=1.8,<2.0` (to allow for seamless upgrade in the future) or remain `django>=1.8,<1.9` and require a separate PR later to finish the update to 1.11? That seems to be what the other repo did, should we do the same here @bmedx?